web/src/i18n/locales/en.yaml

```diff
@@ -356,8 +356,8 @@
     headers:
       procurement: Procurement Standards (Competition / Sole Source)
       recordsAccess: Access to Records, Reporting and Agency Attestations
-      softwareRights: Software & Ownership Rights, Federal Licenses, Information Safeguarding, HIPAA Compliance, and Progress Reports
-      security: Security and interface requirements to be employed for all State HIT systems
+      softwareRights: Software & ownership Rights, federal licenses, information safeguarding, HIPAA compliance, and progress reports.
```

Comment:
These are also titles for sections. I'm OK with them staying as they are. Happy to do a quick English grammar sanity check tomorrow.

Reply:
I reverted many of those changes this morning.